model_authorization_model.go

```diff
@@ -141,7 +134,7 @@ func (o AuthorizationModel) MarshalJSON() ([]byte, error) {
 	if o.Id != nil {
 		toSerialize["id"] = o.Id
 	}
-	if o.SchemaVersion != nil {
+	if true {
```

Comment:
Can remove this check (if true)

Reply:
It's possible, but the generator models will have to be modified - and the effect will be a bit beyond just this file.

Will attempt that when submitting the client PR